website/docs/troubleshooting.md

```diff
@@ -36,3 +36,59 @@
     mixed refresh rate out of the box), if so,that's because X11 does not support mixed refresh
     rate well. You may be able to fix this through your compositor or by switching to wayland.
     As a temporary work around, you may set `g:neovide_refresh_rate` to the lower value.
+
+## Performance Profiling
+
+If you encounter a performance problem like frame rate stuttering, besides attaching a log file when
+reporting bugs, [tracy](https://github.com/wolfpld/tracy) profiling data will also be very useful and
+can usually help developers to troubleshoot the bug much faster. Here is how you can collect tracy data.
+
+1. Install tracy. Windows users can download it at
+[its GitHub release page](https://github.com/wolfpld/tracy/releases). Linux and macOS users can install
+it with package manager. Otherwise, you may have to build it yourself following tracy docs.
+
+2. Install [Rust SDK](https://www.rust-lang.org).
```

Comment:
I think we should also refer to this page https://neovide.dev/installation.html#installation, for more prerequisites for building from source

Reply:
Done!